frontend/packages/component-library/src/actionBlock/ActionBlock.tsx

```diff
@@ -0,0 +1,235 @@
+import classNames from 'classnames';
+import {HTMLAttributes} from 'react';
+
+import {LinkButton} from '@/button';
+import {
+  Heading3,
+  BodyThreeText,
+  OverlineTwoText,
+  StrongText,
+} from '@/typography';
+
+import moduleStyles from './actionBlock.module.scss';
+
+export interface ActionBlockProps extends HTMLAttributes<HTMLDivElement> {
+  /** Action Block title */
+  title: string;
+  /** Action Block description */
+  description: string;
+  /** Action Block image */
+  image?: string;
+  /** Action Block overline */
+  overline?: string;
+  /** Detail label */
+  detailLabel?: string;
+  /** Detail text */
+  detailDescription?: string;
+  /** Primary button label */
+  primaryButtonLabel?: string;
+  /** Primary button link */
+  primaryButtonUrl?: string;
+  /** Primary button aria label */
+  primaryButtonAriaLabel?: string;
+  /** Secondary button label */
+  secondaryButtonLabel?: string;
+  /** Secondary button link */
+  secondaryButtonUrl?: string;
+  /** Secondary button aria label */
+  secondaryButtonAriaLabel?: string;
```

Comment:
I think it will be easier and more consistent to accept full LinkButton props instead of specific props of each button, smth like:
```suggestion
  /** Primary button props */
  primaryButton?: LinkButtonProps;
  /** Secondary button props */
  secondaryButton?: LinkButtonProps;
```

For example, please take a look at [Dialog component](https://github.com/code-dot-org/code-dot-org/blob/3461bab9b3a247e282bad7b328c192195b9b31a6/frontend/packages/component-library/src/dialog/Dialog.tsx)

Reply:
Ooo going to check this out!